guides/github-actions.md

```diff
@@ -0,0 +1,222 @@
+# GitHub Actions recipes
+
+We use [GitHub Actions](https://github.com/features/actions) for some of our contionuous integration and other automations. 
+This guide is a collection of actions recipes to perform tasks that may be common across repositories. 
+
+- [Running front-end unit tests with gulp](#running-front-end-unit-tests-with-gulp)
+- [Running back-end unit tests with tox](#running-back-end-unit-tests-with-tox)
+- [Fetching git history for setuptools-git-version](#fetching-git-history-for-setuptools-git-version)
+- [Attaching a Python wheel file to a GitHub release](#attaching-a-python-wheel-file-to-a-github-release)
+- [Publishing a Python package to PyPI on release](#publishing-a-python-package-to-pypi-on-release)
+- [Publishing MkDocs documentation to GitHub pages](#publishing-mkdocs-documentation-to-github-pages)
+
+
+## Running front-end unit tests 
+
+```yml
```

Comment:
Instead of having the actions duplicated here, maybe link to the [actual file](https://github.com/cfpb/cfgov-refresh/blob/master/.github/workflows/frontend.yml) so we don't have to keep this in sync?

Same for the other snippets in this file.

Reply:
Oh nevermind, I see you say these were tweaked and are for use across projects, not just cfgov-refresh.

Reply:
Yeah... the intention is to provide a starting point for creating actions... I was looking at the ones we have so far, and there are little differences, not huge, but enough that I think they'll need to be tweaked for each repo.